src/app/hooks/use-genres.ts

```diff
@@ -0,0 +1,35 @@
+import { useEffect, useState } from 'react';
```

Comment:
Actually, these hooks are working as data fetching functions. 
It is better to move these into state management like
redux-toolkit - https://redux.js.org/introduction/installation
or
react-query - https://react-query-v3.tanstack.com/installation